Automattic/wp-calypso
# Add support for domain in aftermarket auction status

## Changes proposed in this Pull Request

This PR add support for expired domains in aftermarket auction state, adding notice message and badge in site domains list and domain setting page.

You can apply this diff to your sandbox to mock a domain in auction state D75409-code (_aftermarket-auction-domain.com_).

![auction-list](https://user-images.githubusercontent.com/2797601/155125616-d85c7b39-eb70-4293-905d-761d4b317e56.png)

![auction-detail](https://user-images.githubusercontent.com/2797601/155125632-850fc6aa-85cb-4a5c-a843-ce8ec8f1c052.png)


## Testing instructions

- Build this branch locally or open the live Calypso link
- Select a site and then open site domains list (`upgrades/domains`)
- Verify that the notice message and badge are correctly showed as in the screenshots above
